contracts/helpers/TestPoolCollection.sol

```diff
@@ -62,12 +62,23 @@ contract TestPoolCollection is PoolCollection, TestBlockNumber {
         _poolData[pool].averageRate = newAverageRate;
     }
 
-    function poolWithdrawalAmountsT(Token pool, uint256 poolTokenAmount)
-        external
-        view
-        returns (InternalWithdrawalAmounts memory)
-    {
-        return _poolWithdrawalAmounts(pool, _poolData[pool], poolTokenAmount);
+    function poolWithdrawalAmountsT(
+        Token pool,
+        uint256 poolTokenAmount,
+        uint256 baseTokensWithdrawalAmount
+    ) external view returns (InternalWithdrawalAmounts memory) {
+        Pool storage data = _poolData[pool];
+        PoolLiquidity memory liquidity = data.liquidity;
+
+        return
+            _poolWithdrawalAmounts(
+                pool,
+                poolTokenAmount,
+                baseTokensWithdrawalAmount,
+                liquidity,
```

Comment:
Semantic: you can just put `data.liquidity` here directly.

Reply:
👍